objectivec/GPBMessage.h

```diff
@@ -93,6 +93,10 @@ CF_EXTERN_C_END
 /// @note In DEBUG builds, the parsed message is checked to be sure all required
 ///       fields were provided, and the parse will fail if some are missing.
 ///
+/// @note It is possible the returned errors are also originated in
+///       GPBCodedInputStream and not only in GPBMessage.
```

Comment:
Actually, I guess we don't even point them at the error domain/codes at the top of this file.  Maybe tweak to cover both?  "The errors return are likely from the domains/codes at the top of this file and GPBCodedInputStream.h."…something like that.